system-contract-dapp-playground/__tests__/hedera/hts-interactions/index.test.ts

```diff
@@ -94,6 +101,36 @@ describe('createHederaFungibleToken test suite', () => {
         hash: txHash,
       }),
     }),
+    mintTokenPublic: jest.fn().mockResolvedValue({
```

Comment:
With the `txHash` being the same for all maybe you should add an additional field that confirm the type of transaction.
Maybe similar to the transaction above there should be an event emitted you confirm

Reply:
Agree! I have put in the event filter for the token mint methods. For association and KYC, as the methods emit only ResponseCode events, and the APIs don't utilize these events so I thought I might have leave these out